frontend/src/js/button/PreviewButton.tsx

```diff
@@ -29,7 +29,10 @@ const PreviewButton: FC<PropsT> = ({ url, columns, ...restProps }) => {
   return (
     <Button
       onClick={async () => {
-        await loadPreviewData(getAuthorizedUrl(url), columns);
+        await loadPreviewData(
+          { ...url, url: getAuthorizedUrl(url.url) },
```

Comment:
I don't think we should make this change. Preview doesn't need the `label`, does it? Maybe just keep the `url` prop a `string`?